src/main/resources/application.yaml

```diff
@@ -11,38 +11,77 @@ management:
       exposure:
         include: health, info, prometheus
 
-springdoc:
-  packagesToScan: uk.gov.hmcts.reform.civil.controllers
-  writer-with-order-by-keys: true
+#springdoc:
+#  packagesToScan: uk.gov.moj.sdt.controllers
```

Comment:
Uncomment this part.  Change the packagesToScan value to uk.gov.hmcts.reform.civil.controllers.  Add the writer-with-order-by-keys: true property back after the packagesToScan property.